packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -248,31 +316,28 @@ async function selectPlatformAndExecute(
     target = targetName;
   }
 
-  const mobileCancellationTokenSource = new vscode.CancellationTokenSource();
-  const mobileCancellationToken = mobileCancellationTokenSource.token;
-  const targetUsed = target || platformSelection.defaultTargetName;
-  const command = new SfdxCommandBuilder()
+  const previewCommand = new SfdxCommandBuilder()
     .withDescription(commandName)
     .withArg(sfdxMobilePreviewCommand)
     .withFlag('-p', platformSelection.platformName)
-    .withFlag('-t', targetUsed)
+    .withFlag('-t', target)
     .withFlag('-n', componentName)
     .withFlag('--loglevel', PreviewService.instance.getLogLevel())
     .build();
 
-  const mobileExecutor = new CliCommandExecutor(command, {
+  const previewExecutor = new CliCommandExecutor(previewCommand, {
     env: { SFDX_JSON_TO_STDOUT: 'true' }
   });
-  const execution = mobileExecutor.execute(mobileCancellationToken);
+  const previewExecution = previewExecutor.execute(mobileCancellationToken);
```

Comment:
Should we be reusing this cancellation token?

Reply:
Good point. My rationale was that if this operation gets cancelled then the whole Preview action should be cancelled. But then again, I forgot to check if it indeed got cancelled to exit the function.

So I can either put the check for cancelation here or just keep using 2 different cancelation tokens.

@brandonpage any thoughts?